opal/mca/pmix/pmix2x/Makefile.am

```diff
@@ -10,6 +10,8 @@
 # $HEADER$
 #
 
+if !OPAL_PMIX_USE_EXTERNAL
```

Comment:
I have not thought about this deeply, but it strikes me odd to put a `EXTRA_DIST` inside an automake `if` conditional...?